webview/src/plots/components/ZoomablePlot.tsx

```diff
@@ -73,8 +75,11 @@ export const ZoomablePlot: React.FC<ZoomablePlotProps> = ({
       <button
         className={styles.zoomablePlot}
         onClick={() => handleOnClick()}
+        onMouseDown={() => changeDragAndDropMode(section, dispatch, false)}
```

Comment:
Probably already a known issue that will get worked on later but I currently cannot interact with the plot without activating drag and drop mode:

https://github.com/iterative/vscode-dvc/assets/43496356/d87374ef-2f22-4fcf-ad07-987792da42ae

Reply:
I totally missed that. Thanks for noticing, that was a big fail on my part. It is fixed now.


https://github.com/iterative/vscode-dvc/assets/3683420/a45f4af6-2441-4fd1-b0f7-058f09cff903